build-tests/x86/suse/test-image-disk-ramdisk/appliance.kiwi

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="7.3" name="kiwi-test-image-disk-ramdisk">
+    <description type="system">
+        <author>Marcus Schäfer</author>
+        <contact>ms@suse.com</contact>
+        <specification>Disk test build, deploy to ramdisk</specification>
+    </description>
+    <preferences>
+        <version>1.42.1</version>
+        <packagemanager>zypper</packagemanager>
+        <locale>en_US</locale>
+        <keytable>us</keytable>
+        <timezone>Europe/Berlin</timezone>
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <rpm-check-signatures>false</rpm-check-signatures>
+        <bootsplash-theme>bgrt</bootsplash-theme>
+        <bootloader-theme>openSUSE</bootloader-theme>
+        <type image="oem" filesystem="ext4" firmware="uefi" installiso="true" installboot="install" kernelcmdline="rd.kiwi.ramdisk ramdisk_size=8192000 console=ttyS0">
```

Comment:
`ramdisk_size` parameter and `<size>` section they should match up to my understanding. Otherwise we are wasting memory, right? I'd say a perfect match is the only sane setup as OEM ramdisk images do not expand, so creating a disk image that matches the ramdisk size looks like the best approach to me. So I'd go with 

```
<size unit="G">2</size>
```
and `ramdisk_size=2097152`. This i way smaller and easier to test in host that already has some running VMs and some browser consuming insane amount of memory :)

Also I don't think it makes much sense try  to find out some strategy to allow self expansion on ramdisk deployments. I was thinking about it when we coded the mount hook few months ago. I am not sure it is worth the effort, I'd say use cases are damn limited.

Reply:
> ramdisk_size parameter and <size> section they should match up to my understanding

yes but you can't make them 100% identical because one byte in difference can lead the size checker code to say "not enough space available". Thus ramdisk_size setting should be a bit higher than the image file size. The test here did it absolutely "stable" to avoid non test related problems. But we can go down a bit

Reply:
done now (2G / 2048M image file size) 2192M ramdisk_size .. I don't really care about the ram waste for the purpose of a test, ok ?